pkg/minikube/cluster/cluster.go

```diff
@@ -229,11 +229,38 @@ var assets = []fileToCopy{
 	},
 }
 
-func updateLocalkubeFromURL(config KubernetesConfig, client *ssh.Client) error {
+func (k *KubernetesConfig) getLocalkubeCacheFilepath() string {
+	return filepath.Join(constants.Minipath, "cache", "localkube",
+		filepath.Base("localkube-"+util.GetMD5Hash(k.KubernetesVersion)))
```

Comment:
I think that https://golang.org/pkg/net/url/#QueryEscape may be a good solution.